game_data.py

```diff
@@ -0,0 +1,302 @@
+data = [
```

Comment:
This is good.   For practice, try converting this to JSON and importing it in the Python script.

Reply:
I converted 'game_data.py' into 'game_data.json', imported the file in 'main.py', opened the file, loaded it to a variable, and closed the file at the end of the program

Reply:
Excellent